data_processing/common/custom_logger.py

```diff
@@ -29,13 +31,16 @@ def init_logger(filename='data-processing.log'):
         ch = logging.StreamHandler()
         ch.setLevel(logging.INFO)
         # create formatter and add it to the handlers
-        formatter = MultilineFormatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
         ch.setFormatter(formatter)
         fh.setFormatter(formatter)
         # add the handlers to logger
         logger.addHandler(ch)
         logger.addHandler(fh)
-
+    if logger.handlers:
+        fh = logging.FileHandler(log_file)
+        fh.setLevel(logging.INFO)
+        fh.setFormatter(formatter)
+        logger.addHandler(fh)
```

Comment:
Looks like we have the same fh setup in the other if case. Should we move this outside the ifs?